### ISSUE TITLE
Mic-assignment for Companion App

### ISSUE THREAD

**Issue by m4akus** [UltraStar-Deluxe/Play#407]
Issue type: Feature request

### Actual behaviour

Currently Mics from the Companion app are assigned the same way as regular mics. The upcoming feature of  #297  will help with this but I think there is a way thats a bit easier.

### Expected behaviour

In the Companion App every User can set a Name. Would it be possible if a Player-Name and a "Mic"-Name are the same to automaticlly pre-asign that microphone to that player (should still be able to overwrite the selection, because user may still swap to another mic)

**Comment by achimmihca** [comment]
> Would it be possible if a Player-Name and a "Mic"-Name are the same to automaticlly pre-asign that microphone to that player

yes, I think looking for a mic with matching player name first makes sense in the automatic mic assignment.

This is not a big change.

BTW: in addition to singing into the mic, you will also be able to choose a different mic from a list. Or rather: you can select a different mic from a list either by clicking on it, or by making noise into it.

**Comment by achimmihca** [comment]
This has been implemented for the Steam release
